Comp2023/src/main/java/frc/robot/Constants.java

```diff
@@ -284,9 +284,9 @@
     public static final double kExtensionPidKi            = 0.0;    // Extension PID integral constant
     public static final double kExtensionPidKd            = 0.0;    // Extension PID derivative constant
     public static final int    kExtensionAllowedError     = 0;      // Extension PID allowable closed loop error in counts
-    public static final double kExtensionToleranceInches  = 1;    // Extension PID tolerance in inches
+    public static final double kExtensionToleranceInches  = 1;      // Extension PID tolerance in inches
```

Comment:
As a style recommendation, please try to use floating point numbers for doubles--not integers. It helps the reader.